ios/Redadalertas/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>802628832645-6gbn0jhg1buk34kimru2ifj3h89c62bd.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.802628832645-6gbn0jhg1buk34kimru2ifj3h89c62bd</string>
+	<key>API_KEY</key>
+	<string>AIzaSyC4lz9mPm4JkJNWG91B9GUG2_yDOJpzUII</string>
```

Comment:
I still have a feeling we should keep files like this out of version control, even if it's easily extracted from the APK or whatever. One of the reasons is to prevent people from gaining access to our Firebase, accidentally or otherwise, and also so any developers who want to work on it will be motivated to talk to us to get this information and we can see who they are.

Right now it sounds like it'll be difficult to both do that and have the file available for the distribution build. But long-term we should think about how to handle that. @thejoelpatrol how would this work for the Android distro?

Reply:
I'm fine getting this out of the repo for open source contribution reasons! Like I posted in slack, I would do something like what this script does in order to populate the plist file at build time from ENV variables in App Center:

```
#!/usr/bin/env bash

# Updating Info.plist

PLIST=$BUILD_REPOSITORY_LOCALPATH/MyApp/iOS/Info.plist

/usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString 1.0.${APPCENTER_BUILD_ID}" $PLIST

# Print out file for reference
cat $PLIST

echo "Updated info.plist!"
```

Reply:
Oh ok sweet! I forgot we already talked about this. Maybe you can add that to the second notification PR (and then change the Firebase creds haha)

Reply:
Yeah I guess I could reset whatever credentials I can, but I won't be able to reset all of them (unless we create a totally different Firebase project). I'm happy to do it in this PR so that we at least never merge in the file.

Reply:
Ok, that sounds good for now. We'll see if @thejoelpatrol has any security concerns once you reset what credentials you can.